tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -95,3 +98,24 @@ def test_add_capacity_pre_upgrade(self, reduce_and_resume_cluster_load):
         Test to add variable capacity to the OSD cluster while IOs running
         """
         add_capacity_test()
+
+
+@tier1
+@skipif_no_lso
+@skipif_external_mode
+@skipif_bm
+@skipif_aws_i3
+@skipif_ocs_version("<4.7")
+class TestAddCapacityLso(ManageTest):
```

Comment:
It would be clearer to mention that this is only to be run if flexible scaling is enabled. For instance, an upgrade scenario should not do this as the cluster on earlier versions would not have flexible scaling enabled.